src/components/Overlay/Overlay.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "Overlay",
   "notes": "",
+  "description": "An element that takes over the screen to help bring focus to an element for the user. Used in conjunction with the {0} Panel {1} and {0} Dialog {1} and available in clear and dark.",
```

Comment:
Instead of restarting the count, you have to continue the count so 
{0} Panel {1} and {0} Dialog {1} would become **{0} Panel {1} and {2} Dialog {3}**